src/extension-runners/chromium.js

```diff
@@ -256,6 +256,21 @@ export class ChromiumExtensionRunner {
       chromeFlags.push(...this.params.args);
     }
 
+    // `--remote-debugging-pipe` results in `navigator.webdriver == true` in
+    // the launched Chromium instance (unless a Chromium session exists
+    // already, in which case the existing session is used, which typically
+    // has `navigator.webdriver == false`). This breaks websites with bot
+    // detection features. The following flag prevents this.
+    // Only pass this flag if `--enable-blink-features=AutomationControlled`
+    // is not passed.
+    if (
+      !chromeFlags.some((flag) =>
+        /^--enable-blink-features=(.*,)AutomationControlled(,|)/.test(flag),
```

Comment:
```suggestion
        /^--enable-blink-features=(.*,)?AutomationControlled(,|$)/.test(flag),
```

I realize that this regex is a copy paste of what I sketched before, but it has two mistakes that need to be corrected to have the intended outcome:

- add `?` to make the leading comma optional, test case `--enable-blink-features=AutomationControlled`
- add `$` after the `|` to make sure that we require an anchor. Otherwise something like `--enable-blink-features=AutomationControlledButNotReally` would also be matched.

Reply:
To verify the fix automatically (instead of relying on code review), the unit test could be expanded to test it.

One way to do it is to copy paste the test case twice more, but I'd prefer to not have that level of repetition and reuse the test logic in a new function instead.

The test framework (mocha) has an example showing how this can be done: https://mochajs.org/#dynamically-generating-tests